src/beta.ts

```diff
@@ -0,0 +1 @@
+export * from './index';
```

Comment:
The generated JS is:

```js
__exportStar(require("./index"), exports);
```

Does having tslib installed make that helper available on the global somehow? 

If so, "`or another bundler that supports resource management`" is there any other tool that works, or is this for TS only? (I also see that we say "enable tslib", is there a special import for that?)

Reply:
No, it depends on your ts config settings.  I'm surprised this is what's exported.

Do you have any ideas how we can export the entirety of the index without duplicating the entire file?

Reply:
Does the `export =` solution you have now work? I am unsure but I think this may make this file import strangely if it's a genuine ESM import, but that's something you can manually verify works. 

Another option may be to write `export type * from 'index'` and write the _exportStar logic in TS: essentially `Object.assign(module.exports, require('./index'))`.